fiftyone/core/collections.py

```
@@ -3523,32 +3523,224 @@ def compute_metadata(
             progress=progress,
         )
 
-    def compute_annotation_schema(self, field_name, scan_samples=True):
-        """Compute the annotation schema for a collection's field
+    def generate_label_schema(self, field_name=None, scan_samples=True):
```

Ruff (0.14.7)

Boolean default positional argument in function definition

(FBT002)